### PR TITLE
Hello_Microverse

### PR DESCRIPTION
**Pull request by chidiebereojingwa**
# Hello_Microverse
. Added index file
. Added CSS file
. Added linter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
-# Hello_Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Hello microverse
+
```
Comment (barackm) [inline README.md:4]: - [ ] Can we please add a description of the project in the README, this will help people understand what is the project all about 😉

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @chidiebereojingwa, 👋 

You have done an amazing job so far! 👏👏👏
However, there are some issues that you still need to work on to go to the next project. But remember you are almost there 😉  😉 !

# STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (barackm) [inline README.md:4]: - [ ] It seems that you still don't have a description in your README, you only have the title, can you please add the description? 👍

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @chidiebereojingwa, 👋 

Great work on making the changes requested by a previous reviewer 👏 👏👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 😉  😉 

 # STATUS: Required Changes ♻️

- [ ] Please let's resolve the conflict between the branches

### _Check the comments under the review._ 👇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (chidiebereojingwa) [reply README.md:4]: i added a description after the title. please check
image

Comment (barackm) [reply README.md:4]: Hi @chidiebereojingwa, that's okay but I still can't see it, can you check this file please [https://github.com/chidiebereojingwa/Hello_Microverse/blob/feature-branch/README.md](https://github.com/chidiebereojingwa/Hello_Microverse/blob/feature-branch/README.md)

Comment (chidiebereojingwa) [reply README.md:6]: okay, I have done it now.

Comment (barackm) [reply README.md:6]: That's great 👏 , can you now fix the conflict?

Comment (chidiebereojingwa) [reply README.md:6]: How do I do that? which conflict?

Comment (barackm) [reply README.md:6]: Yeah this is the conflict I am talking about
image

There are some changes you have introduced in the README.md file that created the conflict with the main branch

Comment (chidiebereojingwa) [reply README.md:6]: please I am having a tough time time resolving it can you guide me

**Review by safafa — COMMENTED** [review]
## STATUS: APPROVED 🟢 👏
Hello @chidiebereojingwa  great job working on the required changes 👍 💯
For the upcoming projects, please do not merge the branches before getting your project approved.

HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**